### DISCUSSION TITLE
please do something to have the right click options in android, they are quite necessary, like some mouse simulation

### DISCUSSION THREAD

**Discussion by Chikiak** [godotengine/godot-proposals#5385] [Platforms]
### Describe the project you are working on

Any game

### Describe the problem or limitation you are having in your project

Many options are difficult to use without right clicking, some I haven't even found a way to use.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

add a mouse simulator that allows you to use the right click

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

-

### Is there a reason why this should be core and not an add-on in the asset library?

I don't know, it's just necessary, it doesn't matter how

**Comment by Calinou** [comment]
This was recently added by https://github.com/godotengine/godot/pull/65434 and https://github.com/godotengine/godot/pull/65434, which will be featured in 3.6.
